test/integration-orientdb/bugs/Test_Edge_Creation

```diff
@@ -0,0 +1,120 @@
+var assert = require('assert');
+
+var self = this;
+
+describe('Issue: Found class name null', function () {
+    before(function (done) {
+
+        var fixtures = {
+            AnswerFixture: {
+                tableName: 'Answer',
+                //identity: 'answer',
+
+                attributes: {
+                    id: {
+                        type: 'string',
+                        primaryKey: true,
+                        columnName: '@rid'
+                    },
+                    Answer: {
+                        type: 'JSON'
+                    },
+                    question: {
+                        collection: 'Questions',
+                        via: 'answers',
+                    }
+                }
+            },
+
+            QuestionsFixture: {
+                tableName: 'Questions',
+                //identity:'questions',
+                joinTableNames: {
+                    answers: 'answered'
+                },
+
+                attributes: {
+                    id: {
+                        type: 'string',
+                        primaryKey: true,
+                        columnName: '@rid'
+                    },
+                    question: {
+                        type: 'string'
+                    },
+                    answers: {
+                        collection: 'Answer',
+                        via: 'question',
+                        dominant: true
+                    }
+                }
+            }
+
+        };
+
+        CREATE_TEST_WATERLINE(self, 'test_join_relationship', fixtures, done);
+    });
+    after(function (done) {
+        DELETE_TEST_WATERLINE('test_join_relationship', done);
```

Comment:
Same as above, please use 'test_bug_116' to avoid potential DB conflicts in the future.